src/test/java/uk/gov/hmcts/dts/fact/controllers/admin/AdminCourtAddressControllerTest.java

```diff
@@ -127,53 +125,14 @@ void shouldReturnNotFoundWhenRetrievingAddressesForUnknownCourtSlug() throws Exc
     }
 
     @Test
-    void shouldUpdateCourtAddresses() throws Exception {
```

Comment:
Have these tests been removed?

Reply:
yes they've essentially moved to service tests